moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -53,7 +53,7 @@ export const formatProjectActivity = (change, lookupList) => {
       };
     }
 
-    // if the new field is null or undefined, its because something was removed
+    // if the new field value is null or undefined, its because something was removed
     if (!lookupList[changeData.new[changedField]]) {
```

Comment:
@chiaberry All of this looks great but [you were 100% right](https://github.com/cityofaustin/atd-moped/pull/1311) about [my other PR](https://github.com/cityofaustin/atd-moped/pull/1312/) affecting the way this works. Since `project_name_secondary` now updates to `null` instead of an empty string when removed, removing this field shows as `Changed secondary project name` in the activity log for me.

Not sure if we have other columns that are handled this way but happy to bounce around ideas on this!

Reply:
Does it still say "Changed secondary project name to (none) ?" because if so, i think that works

Reply:
That was the case, but now I see the ` to (none)` part! 🎉